sonic-xcvrd/scripts/xcvrd

```diff
@@ -1157,6 +1161,10 @@ class DaemonXcvrd(DaemonBase):
 
         logger.log_info("Shutting down...")
 
+        if self.sfp_error_event.is_set():
+            logger.log_error("SFP error event occured in SFP monitoring loop")
```

Comment:
I think this log message is unnecessary, since error messages are already being logged at each point that `sfp_error_event.set()`is being called. Suggest to remove.

Reply:
fixed